src/GslCore/PluginTypes.fs

```diff
@@ -88,6 +89,29 @@ type AlleleSwapDesignParams = {
 
 type AlleleSwapProvider = { jobScorer:AlleleSwapJobAccept ; provider:AlleleSwapDesignParams -> GslSourceCode }
 
+
+
+// ==================================================
+// Marker handler for converting ### into sequence
+// ==================================================
+
+type MarkerProviderJobAccept = Capabilities->float<PluginScore> option
+
+/// Information provided when a marker materialization happens
+type MarkerMaterializationTask = { markerSet : string ; dnaSource : string ; ppp:PPP}
+
+type IMarkerProvider =
+    /// Allow marker providers to add command line args and be configurable.
+    inherit IConfigurable<IMarkerProvider>
+
+    /// Marker providers the pragma environment to configure themselves locally.
+    abstract member Setup : PragmaCollection -> IMarkerProvider

```

Comment:
This member is unused; it seems like the current pragma collection should be tucked into the MarkerMaterializationTask type and this member eliminated.